File: Gabriel_Andreas_C#/Mastermind.Tests/EvaluationResult.cs
namespace Mastermind.Tests
{
    public class EvaluationResult
    {
        public EvaluationResult(int corrrectColorCount, int correctPostionCount)
        {
            CorrrectColorCount = corrrectColorCount;
            CorrectPostionCount = correctPostionCount;
        }

        public int CorrrectColorCount { get; set; }
        public int CorrectPostionCount { get; set; }
    }
}

File: Gordan_Maik_Java/EvaluationResult.java
package org.softwerkskammer.magdeburg.mastermind;

public class EvaluationResult {

	private final int wellPlacedColors;
	private final int correctButMissplacedColors;

	public EvaluationResult(int wellPlacedColors, int correctButMissplacedColors) {
		this.wellPlacedColors = wellPlacedColors;
		this.correctButMissplacedColors = correctButMissplacedColors;
	}
	
	
	public int GetWellPlacedColors() {
		return wellPlacedColors;
	}
	
	public int GetCorrectButMissplacedColors() {
		return correctButMissplacedColors;
	}
}
